org.eclipse.january/src/org/eclipse/january/dataset/Slice.java

```diff
@@ -14,60 +14,89 @@
 
 import java.io.Serializable;
 
-/**
- * Class to represent a slice through a single dimension of a multi-dimensional dataset. A slice
- * comprises a starting position, a stopping position (not included) and a stepping size.
+/**	
+ * The {@code Slice} class represents the set of indices (start, stop, step), that are used to extract specifics subsets of {@link org.eclipse.january.dataset.Dataset}.<br><br>
+ * The start argument default to 0, stop argument default to the stop argument default to the end of the dimension that the slice is applied to, and the default argument for the step is 1.
```

Comment:
The correct phrasing in English is:

The start argument defaults to 0
but
and the default argument for the step is 1

 :-)